package.json

```diff
@@ -10,8 +10,8 @@
 		"url": "https://github.com/idanpa/vscode-smartmulticursor"
 	},
 	"bugs": {
-        "url": "https://github.com/idanpa/vscode-smartmulticursor/issues"
-    },
+		"url": "https://github.com/idanpa/vscode-smartmulticursor/issues"
```

Comment:
remove

Reply:
Will remove. I think my editor wanted to align things.

Reply:
Fixed in https://github.com/idanpa/vscode-smartmulticursor/pull/8/commits/51c4b75d76de036382f2ebd8037b00877006e529